src/containers/pages/IRS/plans/tests/index.test.tsx

```diff
@@ -70,15 +71,34 @@ describe('components/IRS Reports/IRSPlansList', () => {
         <IRSPlansList {...props} />
       </Router>
     );
-    await new Promise(resolve => setImmediate(resolve));
-    wrapper.update();
+    await act(async () => {
+      await new Promise(resolve => setImmediate(resolve));
+      wrapper.update();
+    });
     expect(toJson(wrapper.find('BreadcrumbItem li'))).toMatchSnapshot('breadcrumbs');
     expect(toJson(wrapper.find('h3.page-title'))).toMatchSnapshot('page title');
-    expect(toJson(wrapper.find('.thead .tr .th'))).toMatchSnapshot('table headers');
-    expect(toJson(wrapper.find('.tbody .tr .td'))).toMatchSnapshot('table rows');
```

Comment:
Why?

Reply:
> expect(toJson(wrapper.find('.thead .tr .th'))).toMatchSnapshot('table headers');

I believe ^^ is still present.

Reply:
for this `expect(toJson(wrapper.find('.tbody .tr .td'))).toMatchSnapshot('table rows');` the current table we are using is giving large snapshots for this so i opted to remove it and check if the right number of rows were rendered